src/box/lua/slab.c

```diff
@@ -254,6 +254,16 @@ lbox_runtime_info(struct lua_State *L)
 	return 1;
 }
 
+static int
+lbox_runtime_is_recover_complete(struct lua_State *L)
```

Comment:
'recover' is a verb, so correct name of this function should be '...is_recovery_complete(d)'. But personally I'd call it 'is_recovery_done' or 'is_recovery_finished'. Also I bet lua/slab.c is wrong file for such function:) This function has nothing in common with slab allocators (e.g. box/lua/info.c).